### ISSUE TITLE
Consider using the same iframe per origin

### ISSUE THREAD

**Issue by satazor** [eligrey/libxdr#8]
Same origin requests could share the same iframe.
This would reduce the initial delay of the iframe being loaded (because of the additional request that is made).

Is there anything I'm not aware that does not allow that?

Thanks.

**Comment by eligrey** [comment]
Yeah, that's a valid issue. I will check out improving that some day when I'm less busy, or if anyone wants to submit a pullreq I'll gladly merge it.
